src/sync/helpers.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import type { ColumnName } from '..'
+import type { RawRecord } from '../RawRecord'
+
+export function addToRawSet(rawSet: ?string, value: string): string {
+  const array = rawSet ? rawSet.split(',') : []
+  const set = new Set(array)
+  set.add(value)
+  return Array.from(set).join(',')
```

Comment:
Hmm this seems like a lot of work. Maybe we should think about other soultion. But I guess first we need to test it's performance ;) Anyway it's not the most important thing to do right now.

Reply:
this code is unchanged, just moved to a different place ;) I'll be tweaking performance once the whole sync infra is done.

Reply:
Yes, i know just wanted to point that out ;)